build.sbt

```diff
@@ -74,33 +13,43 @@ lazy val root = (project in file("."))
     PgpKeys.publishSigned := {})
 
 lazy val core = (project in file("core"))
-  .enablePlugins(AutomateHeaderPlugin, SbtOsgi, MultiJvmPlugin)
+  .enablePlugins(Common, AutomateHeaderPlugin, SbtOsgi, MultiJvmPlugin)
   .dependsOn(cassandraLauncher % Test)
-  .settings(common: _*)
   .settings(osgiSettings: _*)
   .settings({
-    val silencerVersion = "1.4.1"
     Seq(
       libraryDependencies ++= Seq(
-        compilerPlugin("com.github.ghik" %% "silencer-plugin" % silencerVersion),
-        "com.github.ghik" %% "silencer-lib" % silencerVersion % Provided),
+          compilerPlugin("com.github.ghik" %% "silencer-plugin" % Dependencies.silencerVersion),
+          "com.github.ghik" %% "silencer-lib" % Dependencies.silencerVersion % Provided),
       // Hack because 'provided' dependencies by default are not picked up by the multi-jvm plugin:
-      managedClasspath in MultiJvm ++= (managedClasspath in Compile).value.filter(_.data.name.contains("silencer-lib"))
-      )
+      managedClasspath in MultiJvm ++= (managedClasspath in Compile).value.filter(_.data.name.contains("silencer-lib")))
   })
   .settings(
     name := "akka-persistence-cassandra",
-    libraryDependencies ++= akkaPersistenceCassandraDependencies,
+    libraryDependencies ++= {
+      // Akka Typed is not available for Scala 2.11
+      if (scalaBinaryVersion.value == "2.11") Dependencies.akkaPersistenceCassandraDependencies
+      else Dependencies.akkaPersistenceCassandraDependencies ++ Dependencies.akkaTyped
+    },
+//    Compile / packageBin / packageOptions += Package.ManifestAttributes(
```

Comment:
do we want this?

Reply:
I guess so, but the 0.98 version doesn't contain it. So I left it in for 0.99 to be added.